specifyweb/specify/views.py

```diff
@@ -528,6 +537,59 @@ def update_record(record: models.Model):
 
     # Return http response
     return http.HttpResponse('', status=204)
+
+@app.task(base=LogErrorsTask, bind=True)
+def record_merge_task(self, user_id: int, model_name: str, old_model_ids: List[int], new_model_id: int, merge_id: int,
+                      new_record_dict: Dict[str, Any]=None):
+
+    Collection = getattr(models, 'Collection')
+    Specifyuser = getattr(models, 'Specifyuser')
+    specify_user = Specifyuser.objects.get(id=new_record_dict['specify_user_id'])
+    new_record_info = {
+        'agent_id': new_record_dict['agent_id'],
+        'collection': Collection.objects.get(id=new_record_dict['collection_id']),
+        'specify_user': specify_user,
+        'version': new_record_dict['version'],
+        'new_record_data': new_record_dict['new_record_data']
+    }
+
+    "Run the record merging process as a background task with celery"
```

Comment:
`# should this be a comment?`

or is this ok in python style?

i.e syntatically javascript allows that too, but linters would complain

Reply:
It's the function description, but it must have gotten misplaced somehow, fixed now